src/app/zap-templates/zcl/data-model/chip/window-covering.xml

```diff
@@ -126,11 +126,12 @@ limitations under the License.
   </cluster>
 
   <!-- Window Covering Descriptive Enum Set -->
-  <enum name="WcType" type="ENUM8">
```

Comment:
Please remove `WcType` from the list in `src/app/zap-templates/templates/app/helper.js`.  Same for `WcEndProductType`.

Reply:
wasn't aware of this list in helper.js ; thanks i have just addressed this !